src/main/java/org/isf/medicalstock/gui/MovStockBrowser.java

```diff
@@ -931,7 +937,42 @@ private JButton getDischargeButton() {
 		});
 		return dischargeButton;
 	}
-
+	
+	/**
+	 * this method creates the button that delete the last charge movement 
+	 * 
+	 * @return
+	 */
+	private JButton getDeleteLastMovementButton() {
+		JButton deleteMovementButton = new JButton(MessageBundle.getMessage("angal.common.delete.btn"));
+		deleteMovementButton.setMnemonic(MessageBundle.getMnemonic("angal.common.delete.btn.key"));
+		deleteMovementButton.setMnemonic(KeyEvent.VK_D);
+		deleteMovementButton.addActionListener(new ActionListener() {
```

Comment:
please we are using lambra expression for anonymous classes, and fix the indentation from the next line
```suggestion
		deleteMovementButton.addActionListener(actionEvent -> {
```